src/libs/portfolio/helpers.ts

```diff
@@ -289,17 +290,23 @@ export function getUpdatedHints(
       const isTokenPreference = lowercaseNetworkPreferenceTokenAddresses.includes(lowercaseAddress)
       const isTokenInExternalAPIHints =
         lowercaseERC20HintsFromExternalAPI.includes(lowercaseAddress)
+      const isTokenInToBeLearnedTokens = tobeLearnedTokens.includes(lowercaseAddress)
       const hasBalance = lowercaseNetworkTokenAddressesWithBalance.includes(lowercaseAddress)
 
-      if (!isTokenInExternalAPIHints && !isPinned && !isTokenPreference && hasBalance) {
+      if (
+        !isTokenInExternalAPIHints &&
+        isTokenInToBeLearnedTokens &&
+        !isPinned &&
+        !isTokenPreference &&
+        hasBalance
```

Comment:
This condition is only true the first time based on the logic in the controller. We should update the timestamp every time the portfolio is updated and the token has balance.